sound/soc/sof/topology.c

```diff
@@ -2185,6 +2223,12 @@ static int sof_process_load(struct snd_soc_component *scomp, int index,
 	process->comp.pipeline_id = index;
 	process->config.hdr.size = sizeof(process->config);
 
+	if (process->comp.uid) {
+		dev_dbg(scomp->dev, "tplg: process component, uuid address: 0x%x\n",
```

Comment:
why `uuid address`? The calur is a UUID, right? Does it represent some kind of address? If it does, then maybe "address uuid"?

Reply:
yes, it is 32 bit FW address.

Reply:
@keyonjie I still don't like the "uid", maybe "uuid_key" to imply this is a dictionary key?

Reply:
@kv2019i @lyakh this is already changed as you know, let me mark it resolved.